tests/tensors/constexpr_symmetric_tensor.cc

```diff
@@ -111,6 +111,37 @@ main()
   DEAL_II_CONSTEXPR const auto B = get_tensor_4();
   deallog << "SymmetricTensor<4,2> = " << B << std::endl;
 
+  {
+    DEAL_II_CONSTEXPR double a_init[3][3] = {{1., 0., 0.},
+                                             {2., 1., 0.},
+                                             {3., 2., 1.}};
+    DEAL_II_CONSTEXPR Tensor<2, 3> dummy_a{a_init};
+    DEAL_II_CONSTEXPR SymmetricTensor<2, 3> a = symmetrize(dummy_a);
```

Comment:
```suggestion
    DEAL_II_CONSTEXPR const SymmetricTensor<2, 3> a = symmetrize(dummy_a);
```
Using `const` here is not very critical but it's nice to write. It enforces const-ness even when `DEAL_II_CONSTEXPR` is empty.